### PR TITLE
Typos: fixed in the fixtures README file

### PR DESCRIPTION
**Pull request by kristynrb**
Fixed typos in the description of the organizations.

### CODE REVIEW

**Comment by harsh183** [comment]
Looks good.

File: api/spec/fixtures/README.md [hunk 1]
```diff
@@ -226,12 +226,12 @@ We believe that open source is the future of technology—and that it's time to
 
   - [The Ada Awards](http://adaawards.com/)
 
-  **Description:** The Ada Awards are an initiative of the Digital Leadership Institute, a Brussels-based international NGO with a mission to bridge the gender gap and digital divide worldwide. The Awards aim to recognise outstanding girls and women in tech, and the organisations who support them in Europe, Africa and beyond.
+  **Description:** The Ada Awards are an initiative of the Digital Leadership Institute, a Brussels-based international NGO with a mission to bridge the gender gap and digital divide worldwide. The Awards aim to recognize outstanding girls and women in tech, and the organizations who support them in Europe, Africa and beyond.
```
Comment (cristianoliveira) [inline api/spec/fixtures/README.md]: "organization" and "recognize" - I noticed you are American 🇺🇸 😆

**Comment by cristianoliveira** [comment]
Hey @kristynrb looks good! Although "organisation" and "recognise" aren't wrong in British English it is nice to keep consistency :) 

Thank you!!

**Comment by kristynrb** [comment]
Absolutely! I know those are correct in British English, but I thought that
you'd want them changed for consistency (as you mentioned) :)

**Comment by harsh183** [comment]
Yes, I think we should just standardise on American English.

**Comment by harsh183** [comment]
I mean standardize. 😄

**Pull request merged**